NSS-Day-Cohort-66/rare-client-rare-poultry
# Client Side User Registration Complete

# Client Side Registration View

## How to test

- ```git fetch``` branch ```jrm-rpclient1```
- ```npm run dev```
- head over to registration page and register user
- check database to make sure everything updated correctly

## Completed by -

- Updated Register.jsx
